src/protocols/ethereum/views/TransferSendModal.vue

```diff
@@ -12,7 +12,7 @@
         ref="currentRenderedComponent"
         v-model:transferData="transferData"
         @success="currentStepConfig.onSuccess"
-        @error="(val: any) => error = val"
+        @error="(val) => error = val"
```

Comment:
I don't think this is a reasonable change. When you remove the `any` my IDE is marking the file as bugged (filename is red). Also the line is underlined:
![image](https://github.com/superhero-com/superhero-wallet/assets/8217670/af39db19-15a1-41c9-a977-0b5e5cfa819a)

Reply:
The situation is opposite for me, i'm using different plugin. I can change it back ofc.